cmd/opentelemetry/go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.6.1
 	github.com/uber/jaeger-client-go v2.25.0+incompatible
-	github.com/uber/jaeger-lib v2.2.0+incompatible
+	github.com/uber/jaeger-lib v2.3.1-0.20200925164221-2cac3b34a519+incompatible
```

Comment:
I assume we need a new release?

Reply:
Yeah, that's right

Reply:
https://github.com/jaegertracing/jaeger-lib/releases/tag/v2.4.0

Reply:
shouldn't this now point to 2.4.0?